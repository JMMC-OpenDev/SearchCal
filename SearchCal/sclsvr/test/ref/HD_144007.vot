version https://git-lfs.github.com/spec/v1
oid sha256:7402fea8b1c846bb11ef9c2774d8855924d0c41e8d6f3282201b3e2ebfb26fe3
size 130397
