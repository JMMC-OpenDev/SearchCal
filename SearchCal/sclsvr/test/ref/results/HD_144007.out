version https://git-lfs.github.com/spec/v1
oid sha256:244538f0c0ee7b43e94c63be1628f5d2922d9725b26508ccbb4eaff16475209b
size 55883
